internal/core/domain/round.go

```diff
@@ -323,12 +329,27 @@ func (r *Round) raise(event Event) {
 	r.on(event, false)
 }
 
-func countSweptVtxos(events []Event) int {
-	count := 0
-	for _, event := range events {
-		if e, ok := event.(BatchSwept); ok {
-			count += len(e.LeafVtxos)
+// countBatchAmount sums the amount of Intent receivers, excluding the onchain ones.
+func (r *Round) countBatchAmount() uint64 {
+	amount := uint64(0)
+	for _, intent := range r.Intents {
+		for _, receiver := range intent.Receivers {
+			if receiver.IsOnchain() {
+				continue
+			}
+			amount += receiver.Amount
+		}
+	}
+	return amount
+}
+
+func (r *Round) willBeFullySwept(newBatchSweptLiquiditySwept float64) bool {
```

Comment:
`willBeFullySwept(newBatchSweptLiquiditySwept` guess we're talking about sweeping here? 😂
```suggestion
func (r *Round) isFullySwept(sweptAmountPercentage float64) bool {
```

or
```suggestion
func (r *Round) isFullySwept(sweptAmount uint64) bool {
```